backend/backend/settings/celery/dev.py

```diff
@@ -0,0 +1,2 @@
+from ..deps.ledger import *
+from ..dev import *
```

Comment:
@GuillaumeCisco just curious: what's the goal of the imports here?

Reply:
celery worker/scheduler need the ledger for working correctly, but not `events` nor `node-register` apps which are required for the server.